### PR TITLE
Expert filter enhancement

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -262,21 +262,21 @@
...
   "plannedOutageRate": "Planning outage rate",
   "forcedOutageRate": "Forced outage rate",
   "vlId": "Voltage level ID",
-  "p0": "Constant P",
-  "q0": "Constant Q",
+  "p0": "Constant P (MW)",
+  "q0": "Constant Q (MVar)",
   "loadTapChangingCapabilities": "Ratio on-load",
   "regulatingRatio": "Ratio regulating",
   "ratioTargetV": "Ratio voltage set point",
```
Comment (ayolab) [inline src/translations/en.json]:
```suggestion
  "ratioTargetV": "Ratio voltage set point (kV)",
```

File: src/translations/en.json [hunk 2]
```diff
@@ -262,21 +262,21 @@
   "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage",
   "maxSusceptance": "Maximum available susceptance",
   "SwitchedOnMaxSusceptance": "Switch-on susceptance",
```
Comment (ayolab) [inline src/translations/en.json]:
```suggestion
  "SwitchedOnMaxSusceptance": "Switch-on susceptance (S)",
```

File: src/translations/en.json [hunk 3]
```diff
@@ -262,21 +262,21 @@
   "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage",
   "maxSusceptance": "Maximum available susceptance",
```
Comment (ayolab) [inline src/translations/en.json]:
```suggestion
  "maxSusceptance": "Maximum available susceptance (S)",
```

File: src/translations/en.json [hunk 4]
```diff
@@ -262,21 +262,21 @@
   "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage",
```
Comment (ayolab) [inline src/translations/en.json]: "maxQAtNominalV": "Available Qmax at nominal voltage (MVar)",
```suggestion
  "SwitchedOnMaxQAtNominalV": "Switch-on Q at nominal voltage (MVar)",
```

File: src/translations/en.json [hunk 5]
```diff
@@ -262,21 +262,21 @@
...
   "maxSusceptance": "Maximum available susceptance",
   "SwitchedOnMaxSusceptance": "Switch-on susceptance",
-  "ratedS": "Rated nominal power",
+  "ratedS": "Rated nominal power (MW)",
```
Comment (ayolab) [inline src/translations/en.json]:
```suggestion
  "ratedS": "Rated nominal power (MVA)",
```

**Review by ayolab — CHANGES_REQUESTED** [review]

**Review by anistouri — APPROVED** [review]

File: src/translations/en.json (hunk 5 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done

File: src/translations/en.json (hunk 4 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done

File: src/translations/en.json (hunk 3 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done

File: src/translations/en.json (hunk 2 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done

File: src/translations/en.json (hunk 1 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done

**Review by ayolab — APPROVED** [review]

**Pull request merged**